app/models/profile.rb

```diff
@@ -106,6 +106,8 @@ def clone_to(account: nil, host: nil)
     new_profile = in_account(account)
     if new_profile.nil?
       (new_profile = dup).update!(account: account, hosts: [host])
+    else
+      new_profile.hosts << host unless new_profile.hosts.include?(host)
```

Comment:
` unless new_profile.hosts.include?(host)` is required in order not to fail with `ActiveRecord::RecordInvalid: Validation failed: Host has already been taken`